src/cmds/core/other.py

```diff
@@ -46,7 +63,15 @@ async def spoiler(self, ctx: ApplicationContext, url: str) -> Interaction | Webh
         channel = self.bot.get_channel(settings.channels.SPOILER)
         await channel.send(embed=embed)
         return await ctx.respond("Thanks for the reporting the spoiler.", ephemeral=True, delete_after=15)
+    
+    @slash_command(guild_ids=settings.guild_ids, description="Provide feedback to HTB!")
+    @commands.cooldown(1, 60, commands.BucketType.user)
+    async def feedback(self, ctx: ApplicationContext) -> Message:
+        """ Provide Feedback to HTB  """
+        modal = FeedbackModal(title="Feedback) # Send the Modal defined above in Feedback Modal, which handles the callback
```

Comment:
Missing closing double-quote `"`
```suggestion
        modal = FeedbackModal(title="Feedback") # Send the Modal defined above in Feedback Modal, which handles the callback
```